src/domain/sales-channels/pages/details.tsx

```diff
@@ -0,0 +1,458 @@
+
+  const openUpdateModal = () => setShowUpdateModal(true)
+  const closeUpdateModal = () => setShowUpdateModal(false)
+
+  return (
+    <div className="col-span-2 rounded-rounded border bg-grey-0 border-grey-20 px-8 py-6">
+      <SalesChannelDetailsHeader
+        resetDetails={resetDetails}
+        salesChannel={salesChannel}
+        openUpdateModal={openUpdateModal}
+        setShowAddProducts={setShowAddProducts}
+      />
+
+      <SalesChannelProductsTable
+        salesChannelId={salesChannel.id}
+        showAddModal={() => setShowAddProducts(true)}
+      />
+
+      {showUpdateModal && (
+        <EditSalesChannel
+          handleClose={closeUpdateModal}
+          salesChannel={salesChannel}
+        />
+      )}
+
+      {showAddProducts && (
+        <SalesChannelProductsSelectModal
+          salesChannel={salesChannel}
+          handleClose={() => setShowAddProducts(false)}
+        />
+      )}
+    </div>
+  )
+}
+
+/**
+ * Sales channels details page container.
+ */
+function Details() {
+  const [filterText, setFilterText] = useState<string>()
+  const [showCreateModal, setShowCreateModal] = useState(false)
+
+  const [
+    activeSalesChannel,
+    setActiveSalesChannel,
+  ] = useState<SalesChannel | null>()
+
+  const { store } = useAdminStore()
+  const { sales_channels } = useAdminSalesChannels()
+
+  useEffect(() => {
+    if (sales_channels && store) {
+      if (!activeSalesChannel) {
+        setActiveSalesChannel(
+          sales_channels.find((sc) => sc.id === store.default_sales_channel_id)
+        )
+      } else {
+        setActiveSalesChannel(
+          sales_channels.find((sc) => sc.id === activeSalesChannel.id)
+        )
+      }
+    }
+  }, [sales_channels, store, activeSalesChannel?.id])
+
+  const openCreateModal = () => setShowCreateModal(true)
+  const closeCreateModal = () => setShowCreateModal(false)
+
+  const resetDetails = () => {
+    setActiveSalesChannel(
+      sales_channels.find((sc) => sc.id === store?.default_sales_channel_id)
+    )
+  }
+
+  function defaultChannelsSorter(sc1, sc2) {
+    if (sc1.id === store?.default_sales_channel_id) {
+      return -1
+    }
+    if (sc2.id === store?.default_sales_channel_id) {
+      return 1
+    }
+
+    return sc1.name.localeCompare(sc2.name)
+  }
+
+  function filterSalesChannels(channels: SalesChannel[]) {
+    if (!filterText) {
+      return channels
+    }
+
+    return channels.filter((ch) => {
+      const filter = filterText.toLowerCase()
+      return (
+        !!ch.name.toLowerCase().match(filter) ||
+        !!ch.description?.toLowerCase().match(filter)
+      )
+    })
+  }
+
+  if (!sales_channels || !activeSalesChannel) {
+    return null
+  }
+
+  return (
+    <div>
+      <Link to={"/a/settings"}>
+        <div className="flex text-grey-40 hover:text-purple-600 gap-2 items-center mb-2">
+          <ArrowLeftIcon />
+          <span className="text-small">Back to Settings</span>
+        </div>
+      </Link>
+
```

Comment:
Issue: use the breadcrumb component for consistency. 

The design of the component has been updated since we introduced it, but we should still use the component so it will be easier to change all breadcrumbs at once later.

Reply:
agree